perceval/backends/github.py

```diff
@@ -39,11 +40,24 @@
 
 GITHUB_URL = "https://github.com/"
 GITHUB_API_URL = "https://api.github.com"
-
+RATE_LIMIT_MIN = 10  # Min rate limit before sleeping until rate limit reset
```

Comment:
I think `MIN` should be a prefix not a suffix, as we do with `MAX_ISSUES` or similar constants.